app.d.ts

```diff
@@ -0,0 +1,27 @@
+import type { OutgoingHttpHeader } from "http"
+
+declare global {
+  type Optional<T> = T | null
```

Comment:
Where is this used? Could it be replaced with fp-ts Optiona type? Affraid these two could be confused now.